x-pack/solutions/security/packages/kbn-cloud-security-posture/public/src/components/fleet_extensions/azure_credentials_form/azure_hooks.ts

```diff
@@ -129,7 +129,7 @@ export const useAzureCredentialsForm = ({
 
   useEffect(() => {
     const isInvalid = setupFormat === AZURE_SETUP_FORMAT.ARM_TEMPLATE && !hasArmTemplateUrl;
-    if (isInvalid !== isValid) {
+    if (isInvalid && isValid) {
```

Comment:
This is confusing. Can we name `isInvalid` somehow differently or change the bool logic somehow to make it clearer? Otherwise it's like "if true and false at the same time, then..."